cli/cmd/testdata/deployment_test_file.golden.yml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apps/v1beta1
```

Comment:
I think this file should start with the test name, "TestInjectYAML", according to, e.g., https://medium.com/soon-london/testing-with-golden-files-in-go-7fccc71c43d3.

Also, it is likely that we'll have multiple tests for this, so maybe include "emojivoto" in the name.

I don't think "test_file" is a useful part of the filename. I suggest dropping it.

Wrapping up, I suggest this file be named "TestInjectYAML_emojivoto_deployment.golden.yml"